cheatsheets/Secrets_Management_Cheat_Sheet.md

```diff
@@ -96,24 +96,15 @@ generally is much more difficult.
 - **Structures and Classes:** In .NET and Java, do not use immutable structures
```

Comment:
Not in this PR, since it had a specific purpose, but I wonder if we eventually we should remind developers that also avoid treating secretes with immutable types such as strings is not even possible without rewriting significant portions of the standard SDKs. (E.g.,the Jakarta Servlet API is a good example. All the parameter values are returned as Strings.) [Note: I'm not volunteering to do that PR, just asking thoughts if whether we should consider it.] Sometimes it keeping secrets confidential is the primary goal, maybe one should favor C/C++ over languages that are garbage collected. Sometimes you have to choose your poison.